Assets/Scripts/Timing/UITiming.cs

```diff
@@ -59,7 +59,14 @@ public class UITiming : MonoBehaviour {
             t.localPosition = new Vector3(0, position.y, position.z);
             window.alpha = 0;
 
-            string ffmpegPath = Path.Combine(Application.streamingAssetsPath, "FFMPEG/ffmpeg.exe");
+            string ffmpegPath = Path.Combine(Application.streamingAssetsPath, "FFMPEG", "ffmpeg.exe");
```

Comment:
Making a helper function to get this path would be helpful, especially since there's actually more logic based on os.

Reply:
Apparently running bash/zsh within unity didn't give it the same PATH as it normally would if you were to run it in a terminal, was forced to use a shell script which added the necessary path where binaries usually are located on linux/mac.